SlyCaptainFlint/react-native
# Create test_workflow.txt

## Summary

Testing out the new PR closer workflow

## Changelog



[CATEGORY] [TYPE] - Message

## Test Plan

1. Create a PR
2. Close the PR by pushing a commit with a message as described [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). This closes the PR, but leaves it with an umerged status.
3. Observe that the new close_pr.yml action will post a comment on the PR with the name of the commit author and the merge commit sha, and label the PR with "Merged"
